src/pages/instances/InstanceOverview.tsx

```diff
@@ -49,10 +49,10 @@ const InstanceOverview: FC<Props> = ({ instance }) => {
                 <th className="u-text--muted">Base image</th>
                 <td>
                   {instance.config["image.description"] ? (
-                    <ResourceLabel
-                      type="image"
-                      value={instance.config["image.description"]}
-                    />
+                    <>
+                      <ResourceLabel type="image" value="" />
+                      {instance.config["image.description"]}
```

Comment:
oh interesting, I see what you did here. Maybe we should truncate the image description here in case if it gets too long?

Reply:
This is not the cleanest way to do it. Maybe we should add a "normal font size" option to the Resource Label component instead. So it gives us truncation and it might be useful in other places.

Reply:
We can always use the regular font, I think.

Reply:
If we use normal font size for `ResourceLabel`, then in notifications the font size will be different between `ResourceLabel` and `ResourceLink` (see image below). Although I think it actually looks okay given that `ResourceLabel` usually appear next to text with normal font size. wdyt?

![Screenshot from 2024-10-24 09-39-45](https://github.com/user-attachments/assets/ef72569b-c2a4-4b7b-95a5-0cf4bbfce489)

Reply:
Yeah, I think this looks all right.